src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/RuntimeBinderException.cs

```diff
@@ -30,6 +42,12 @@ public RuntimeBinderException(string message)
         {
         }
 
+        internal RuntimeBinderException(ErrorCode code, string message)
```

Comment:
I think we may not have access to internal members via reflection in some situations. That is likely the reason the whole class is public. This might need to be public.